helm/hpcc/templates/_helpers.tpl

```diff
@@ -73,13 +73,13 @@ esp:
 secretTimeout: {{ .Values.secrets.timeout | default 300 }}
 storage:
   daliPlane: {{ $daliStoragePlane }}
-  dllsPlane: {{ $daliStoragePlane }}
+  dllsPlane: {{ $dllStoragePlane }}
   dataPlane: {{ $dataStoragePlane }}
   spillPlane: {{ $spillStoragePlane }}
   planes:
 {{- /*Generate entries for each data plane (removing the pvc).  Exclude the planes used for dlls and dali.*/ -}}
 {{- range $plane := $planes -}}
- {{- if and (ne $plane.name $daliStoragePlane) (ne $plane.name $dllStoragePlane) }}
+ {{- if has "data" $plane.labels }}
```

Comment:
this will break any existing helm chart that explicitly defines planes.  @richardkchapman is that ok?  At a minimum it needs clear documentation, and (if needed) extending the schema for 7.12.x so the labels can be added ahead of time.

Reply:
I think it be better if planes defaulted to data (probably the most common) and the test was

    if or (eq 0 (len $planes.labels)) (has "data" $planes.labels)